Wrappers/Python/conda-recipe/meta.yaml

```diff
@@ -32,6 +32,7 @@ requirements:
     - eqt
     - importlib_metadata    # [py<38]
     - h5py
+    - schema
```

Comment:
Do we really need `schema`? Can we just do without validation? The user is responsible for passing the right stuff?

Reply:
I think it makes it more user-friendly